### DISCUSSION TITLE
Current Time

### DISCUSSION THREAD

**Discussion by stevoboy** [Zefau/ioBroker.jarvis#2857] [Q&A]
Dear all, i am using the bar chart to display hourly energy prices.
Is it possible somehow to highlight the bar, which relates to the current time?
I refresh Jarvis via iobroker Hourly.

**Comment by mcuiobroker** [comment]
Which adapter tibber, awattar?
In jarvis not yet.
Use flexChart and iFrame(jarvis)

**Reply by stevoboy** [reply]
Tibber. 
Do the other options integrate in Jarvis?

**Comment by mcuiobroker** [comment]
There is an advanced program, not jarvis internally, to use tibber-Data as stateHTML with highlight hourly. 
Take a look.
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/tibber/tibberhtml-pro

**Reply by stevoboy** [reply]
Looks Like you need the Pro Version of Jarvis. I dont have that and wouldn't want to pay for that. 
Maybe EVCC can display the prices in Jarvis?
